src/components/downloadDocumentButton/index.tsx

```diff
@@ -2,21 +2,45 @@ import React from 'react';
 import DownloadIcon from '@mui/icons-material/Download';
 import { IconButton } from '@mui/material';
 import styled from 'styled-components';
+import config from 'config/index';
 
 interface DownloadDocumentButtonProps {
   fileUrl: string;
 }
 
 const DownloadDocumentButton: React.FC<DownloadDocumentButtonProps> = ({ fileUrl }) => {
+  const rawPdfDownloaderUrl = `${config.BASE_URL}document/download-pdf/${fileUrl}`;
```

Comment:
please use the same approach as for the attachments download(follow the exact same approach)
Create a service for the http request logic.